release notes/v0.35.0.md

```diff
@@ -0,0 +1,94 @@
+k6 v0.35.0 is here! :tada: It introduces several new features that nicely enhance usage of k6 as well as contains a whole number of fixes and ongoing efforts with refactoring.
+
+In total, we have closed 14 issues and merged 43 PRs. We have also branched out three new xk6 extensions during this release :star:
+
+## New features
+
+### VU tags ([#2172](https://github.com/grafana/k6/pull/2172))
+
+k6 now supports setting tags for VUs as part of the Execution API with an easy key-value interface. Example usage:
+
+```js
+import exec from 'k6/execution';
+import { sleep } from 'k6';
+
+export const options = {
+    duration: '10s',
+    vus: 3,
+};
+
+export default function () {
+    exec.vu.tags['mytag'] = 'value';
+    exec.vu.tags['vuId'] = exec.vu.idInTest;
+    console.log(`mytag is ${exec.vu.tags['mytag']} and my VU's ID in tags ${exec.vu.tags['vuId']}`);
+    sleep(5);
+}
+```
+
+There are some caveats, however; using `Object` or `Array` as a value will result in a warning or an exception if `--throw` option is used. Additionally, given that k6 has a whole bunch of system tags, one should be careful with using them as keys. You can read complete information about VU tags in [`k6/execution docs`](https://k6.io/docs/javascript-api/k6-execution/#vu).
+
+### JS promises are now an option 
+
+With the goja update in [#2197](https://github.com/grafana/k6/pull/2197), you can now make a `Promise` and chaining in your k6 scripts:
+
+```js
+export default function () {
+    var p = new Promise((resolve, reject) => {
+        console.log('do something promising!');
+        reject('here');
+    });
+
+    p.then(
+        (s) => { console.log('fulfilled with', s) },
+        (s) => { console.log('rejected with', s) },
+    );
+}
+```
+
+### InfluxDB v2 API
+
+After hard work at working out how to integrate InfluxDB v2 API, it was decided to pull that integration into an [xk6-output-influxdb extension](https://github.com/grafana/xk6-output-influxdb). InfluxDB v1 API is still available as before, with minor optimizations ([#2190](https://github.com/grafana/k6/pull/2190)).
+
+### New xk6 extensions
+
+With this release, we have added two brand-new xk6 extensions:
+
+- [xk6-browser](https://github.com/grafana/xk6-browser) is a browser automation extension which relies on [Chrome Devtools Protocol](https://chromedevtools.github.io/devtools-protocol/).
+- [xk6-output-remote-write](https://github.com/grafana/xk6-output-prometheus-remote) is an output extension with support for [Prometheus Remote-Write protocol](https://docs.google.com/document/d/1LPhVRSFkGNSuU1fBd81ulhsCPR4hkSZyyBj1SZ8fWOM/edit#heading=h.3p42p5s8n0ui).
+
+### Other notable changes
+
+- k6's GRPC was extended with a support for server reflection which allows one to use GRPC even without `proto` file at hand. You can read more about the protocol [here](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) ([#2160](https://github.com/grafana/k6/pull/2160)).
```

Comment:
we in general thank the people (generally authors) that helped with a feature at the end of the line after the PR number for [example](https://github.com/grafana/k6/blob/v0.34.0/release%20notes/v0.33.0.md?plain=1#L34)

Reply:
we also do that even if they did not make the final PR, they still did help and in this particular case AFAIK they did 99% of the work

Reply:
I wasn't the reviewer for that PR and didn't realize it was 99% :confused: This suggests that this way of writing release notes may not be the best TBH.